Automattic/jetpack
# AI Extension: Enhance blocks parsing flickering with child blocks

Fixes #32516
Test with D119992-code

## Proposed changes:

* Changes the way the blocks are verified to avoid flickering and multiple history entries with invalid child blocks

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test with D119992-code
* Check that fields with children, like multiple choice inputs, do not flicker when updated and only one history change is added when a new child is inserted
